numpy/polynomial/_polybase.py

```diff
@@ -144,10 +144,23 @@ def _line(off, scl):
     def _roots(c):
         pass
 
-    @staticmethod
-    @abc.abstractmethod
-    def _fromroots(r):
-        pass
+    @classmethod
+    def _fromroots(cls, roots):
+        if len(roots) == 0:
+            return np.ones(1)
+        else:
+            [roots] = pu.as_series([roots], trim=False)
+            roots.sort()
+            p = [cls._line(-r, 1) for r in roots]
+            n = len(p)
+            while n > 1:
+                m, r = divmod(n, 2)
+                tmp = [cls._mul(p[i], p[i+m]) for i in range(m)]
+                if r:
+                    tmp[0] = cls._mul(tmp[0], p[-1])
```

Comment:
Copied directly from one of the implementations, only change is `*mul` &rarr; `cls._mul`